apps/vh/demo/base/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+namespace: vh
+patchesStrategicMerge:
+  - ../../identity/vh-aad-identity.yaml
+  - ../../identity/demo.yaml
+  - ../../user-api/demo.yaml
```

Comment:
Need to add these files for the kustomize script to pass, you can add them to this PR or remove these patches and add them in subsequent PRs